scripts/testserver_codegen.sh

```diff
@@ -10,6 +10,5 @@ do
     autorest --input-file=./node_modules/@microsoft.azure/autorest.testserver/swagger/$i.json --output-folder=./test/integration/generated/$i --namespace=$i --use=.
     cd ./test/integration/generated/$i
     swift build
-    swift package generate-xcodeproj
     cd ../../../..
```

Comment:
This can put you in a wonky state if the `testserver_codegen.sh` script fails. Is there any way to ensure we get back to the original starting directory?

Reply:
I need to investigate how we can do that. Will be address in other PR.